src/assets/styles/base.scss

```diff
@@ -156,6 +156,11 @@ p {
   font-weight: normal;
 }
 
+.small-text {
```

Comment:
Instead, how about overriding some of the existing Vuetify classes (so we can just the existing classes)?

See https://vuetifyjs.com/en/styles/text-and-typography/#typography

Re: https://github.com/bcgov/namerequest/blob/cd01435b6ddb4f003ebe1ace8593e7ec483c1ef8/client/src/assets/scss/overrides.scss#L13
See

Reply:
I'll look into it, but if the scope strays too far out globally it will deserve it's own ticket.
I'm much over time on these two tickets with continual style/ux iterations.

This class is used in one instance that i know about for now, so i'll move it to it's local component until we know it's going to be something more common.

Reply:
Updated in next commit. Followed your recommendation and overrode `text-body-3` to accommodate our small size here.